### DISCUSSION TITLE
恢复视频评论区右上角的页码跳转区

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1620] [功能建议 / Ideas]
可参见隔壁：https://github.com/MotooriKashin/Bilibili-Old/issues/34#issuecomment-727653104

这玩意 B 站偷偷砍掉好几个月了都没有加回来的意思，也去反馈论坛反馈过。

Bilibili Evolved 也集成一下？

**Comment by the1812** [comment]
还有这种事情? 我都没发现(

**Reply by Geeyun-JY3** [reply]
确实挺难发现的，我当时要不是逛逛隔壁更新日志也没发现 ~~（果然是个砍了才知道有的功能~~

**Comment by Geeyun-JY3** [comment]
行了现在评论区改版后，这个需求又和 #473 一样靠这个脚本解决了：https://greasyfork.org/scripts/403348
